integration/exec_test.go

```diff
@@ -714,3 +714,24 @@ func TestSystemProperties(t *testing.T) {
 		t.Fatalf("kernel.shmmni property expected to be 8192, but is %s", shmmniOutput)
 	}
 }
+
+func TestSeccompNoSleep(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	rootfs, err := newRootfs()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer remove(rootfs)
+	config := newTemplateConfig(rootfs)
+	config.Seccomp = &configs.Seccomp{}
+	config.Seccomp.Syscalls = append(config.Seccomp.Syscalls, &configs.Syscall{
+		Value:  syscall.SYS_NANOSLEEP,
+		Action: configs.Action(syscall.EPERM),
+	})
+	_, _, err = runContainer(config, "", "/bin/sh", "-c", "sleep 2")
+	if err == nil {
```

Comment:
Makes sense to check error text. We had test with just wrong cmd syntax for years in docker :)